app/service/service_senders_schema.py

```diff
@@ -0,0 +1,11 @@
+add_service_email_reply_to_request = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "POST service email reply to address",
+    "type": "object",
+    "title": "Add new email reply to address for service",
+    "properties": {
+        "email_address": {"type": "string", "format": "email_address"},
+        "is_default": {"type": "boolean"}
```

Comment:
Cann you mark this as optional?

Reply:
jsonschema knows it's optional because it's not in the `required` list below

Reply:
I think this is ok because we assume that if it is not included in the schema then `is_default`=True

Reply:
is_default doesn't need to be optional on internal method now?